### PR TITLE
Pass Rt through API

### PR DESCRIPTION
**Pull request by igorkofman**
This PR addresses issue https://github.com/covid-projections/covid-data-model/issues/< >

### Please Check
- [ ] Will the *current* schema in [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.md) be updated with this PR?
- [ ] Are tests passing?

### CODE REVIEW

File: libs/datasets/can_model_output_schema.py [hunk 1]
```diff
@@ -32,13 +34,13 @@
     DEAD,
     BEDS,
     CUMULATIVE_INFECTED,
-    "j",
-    "k",
+    R_T,
+    R_T_STDEV,
```
Comment (ghop02) [inline libs/datasets/can_model_output_schema.py:38]: running out of letters! :)

File: libs/functions/calculate_projections.py [hunk 2]
```diff
@@ -80,6 +82,8 @@ def _calculate_projection_data(file_path):
         peak_hospitalizations_short_falls = df.iloc[df.all_hospitalized.idxmax()].short_fall
         peak_deaths_date = df.iloc[df.new_deaths.idxmax()].date
         population = df.iloc[0].population
+        r_t = df.iloc[10].r_t
```
Comment (ghop02) [inline libs/functions/calculate_projections.py:85]: why 10?

File: libs/datasets/projections_schema.py [hunk 3]
```diff
@@ -31,6 +33,8 @@
     "Peak Hospitlizations Shortfall",
     "Beds at Peak Hospitilization Date",
     "Population"
+    "Rt"
+    "Rt_stdev"
```
Comment (goldblatt) [inline libs/datasets/projections_schema.py:37]: I think these need commas :)

Comment (igorkofman) [reply libs/datasets/projections_schema.py:37]: ty!

**Pull request merged**